langchain4j-bedrock/src/test/java/dev/langchain4j/model/bedrock/BedrockEmbeddingIT.java

```diff
@@ -83,4 +83,70 @@ void testBedrockTitanEmbeddingModelV2() {
         assertThat(embeddingModel.dimension()).isEqualTo(256);
     }
 
+    @Test
+    void testBedrockCohereEmbedEnglishTextV3() {
```

Comment:
Does these tests work for you? I am getting the following error:

```
java.lang.IllegalArgumentException: vector cannot be null

	at dev.langchain4j.internal.Exceptions.illegalArgument(Exceptions.java:19)
	at dev.langchain4j.internal.ValidationUtils.ensureNotNull(ValidationUtils.java:50)
	at dev.langchain4j.internal.ValidationUtils.ensureNotNull(ValidationUtils.java:37)
	at dev.langchain4j.data.embedding.Embedding.<init>(Embedding.java:25)
	at dev.langchain4j.model.bedrock.BedrockCohereEmbeddingResponse.toEmbedding(BedrockCohereEmbeddingResponse.java:27)
	at dev.langchain4j.model.bedrock.internal.AbstractBedrockEmbeddingModel.embedAll(AbstractBedrockEmbeddingModel.java:56)
	at dev.langchain4j.model.bedrock.BedrockEmbeddingIT.testBedrockCohereEmbedMultilingualTextV3(BedrockEmbeddingIT.java:133)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
```

Reply:
I've corrected it. Made Access Request to AWS Bedrock for both and now all green.
<img width="458" alt="Screenshot 2024-11-02 at 18 41 42" src="https://github.com/user-attachments/assets/9972b6f2-1dec-4228-b75c-341dcc350993">